prettier/prettier-check.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: prettier-check
```

Comment:
nit: rename this from `prettier-check`  -> `prettier` since this might do more than check.